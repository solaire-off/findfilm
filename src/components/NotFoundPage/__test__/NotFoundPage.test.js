import React from "react";
import { render } from "@testing-library/react";
import { NotFoundPage } from "..";
import { MemoryRouter } from "react-router-dom";

it("Current render", () => {
  const { asFragment } = render(
    <MemoryRouter initialEntries={["/wrong-url"]}>
      <NotFoundPage />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
