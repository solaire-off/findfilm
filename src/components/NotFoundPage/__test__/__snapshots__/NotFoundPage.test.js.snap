// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current render 1`] = `
<DocumentFragment>
  <div
    class="fullpage text-center"
  >
    <h1
      class="fullpage__title"
    >
      Page not found
    </h1>
    <a
      href="/"
    >
      <button
        class="btn btn--outline-danger  "
        type="button"
      >
        Homepage
      </button>
    </a>
  </div>
</DocumentFragment>
`;
